src/views/CVEdit/CVEdit.tsx

```diff
@@ -28,9 +28,17 @@ export const CVEdit = () => {
 
     const sendForm = async (e: SyntheticEvent) => {
         e.preventDefault();
-        if ((form.firstName==='')||(form.lastName==='')||(form.githubUsername==='')||(form.projectUrls==='')){
+        let validproject = false;
+        form.projectUrls.split(' ').forEach(el => {
+            if (!/^(ftp|http|https):\/\/[^ "]+$/.test(el)) {
+                validproject = true;
+            }
+        });
+        console.log(form.firstName, form.lastName, form.githubUsername, form.projectUrls, validproject);
```

Comment:
jesli consol.log nie potrzebny to do usunięcia

Reply:
Usunąłem